Backend/src/test/java/com/onbank/ftp/FtpClientTest.java

```diff
@@ -0,0 +1,71 @@
+package com.onbank.ftp;
+
+import com.onbank.LoadProperties;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockftpserver.fake.FakeFtpServer;
+import org.mockftpserver.fake.UserAccount;
+import org.mockftpserver.fake.filesystem.DirectoryEntry;
+import org.mockftpserver.fake.filesystem.FileEntry;
+import org.mockftpserver.fake.filesystem.FileSystem;
+import org.mockftpserver.fake.filesystem.UnixFakeFileSystem;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+
+import java.io.File;
+import java.io.IOException;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@LoadProperties
+@SpringBootTest
+class FtpClientTest {
+
+    private FakeFtpServer fakeFtpServer;
+
+    @Autowired
+    private FtpConnection ftpConnection;
+
+    @BeforeEach
```

Comment:
Zmieńmy na beforeAll jeśli ten sam ftp jest używany we wszystkich testach, zaoszczędzi nam to czas